public/js/all.js

```diff
@@ -0,0 +1,242 @@
+/*eslint-disable*/
+document.getElementById('next').addEventListener('click', next);
+document.getElementById('back').addEventListener('click', back);
+document.getElementById('play').addEventListener('click', play);
+document.onkeydown = checkKey;
+
+const urlParams = new URLSearchParams(window.location.search);
+const url = '/visualize_all_json';
+var i = 0;
+var size;
+var len;
+var simulation;
+var result;
+var group;
```

Comment:
these var can be on the same line for less lines check if it works after it

Reply:
done